localization/english/culture/wc_base_innovations_l_english.yml

```
@@ -0,0 +1,68 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ 
+ #####################
+ # Late Medieval Era #
+ #####################
+ 
+ #####################
+ # Early Modern Era  #
+ #####################
+ 
+ #Mil
+ innovation_conscripts:0 "Conscripts"
+ innovation_conscripts_desc:0 "Conscription laws allow us to more field a larger and cheaper army."
```

Removed a unnecessary word.
```suggestion
 innovation_conscripts_desc:0 "Conscription laws allow us to field a larger and cheaper army."
```